webclient/package.json

```diff
@@ -1,20 +1,25 @@
 {
-  "name": "agave-webclient",
-  "version": "1.0.0",
+  "name": "@aics/agave-webclient",
+  "version": "1.5.0",
```

Comment:
Is it possible to have this auto-update with new release versions, or does this have to be manually updated?

Reply:
there's a version bump script (tbump.toml) that goes to all the files that have these version number in them and updates all of them